### PR TITLE
Added default configurations for attributes

### PR DESCRIPTION
**Pull request by dgon-jd**

### CODE REVIEW

File: CloudFabric.EAV.Domain/Models/Attributes/TextAttributeConfiguration.cs [hunk 1]
```diff
@@ -26,15 +34,26 @@ public TextAttributeConfiguration(IEnumerable<IEvent> events) : base(events)
...
+        new List<LocalizedString>
+        {
+            LocalizedString.English("Number")
+        },
+        "text",
+        100,
+        false,
+        new List<LocalizedString>
+        {
+            LocalizedString.English("Price")
```
Comment (ustims) [inline CloudFabric.EAV.Domain/Models/Attributes/TextAttributeConfiguration.cs:48]: This is a text attribute, is "price" supposed to be here?

File: CloudFabric.EAV.Domain/Models/Attributes/TextAttributeConfiguration.cs [hunk 2]
```diff
@@ -26,15 +34,26 @@ public TextAttributeConfiguration(IEnumerable<IEvent> events) : base(events)
     {
         Apply(new TextAttributeConfigurationUpdated(id, defaultValue, maxLength, isSearchable));
     }
+    public TextAttributeConfiguration(string machineName, Guid? tenantId) : this(Guid.NewGuid(),
+        machineName,
+        new List<LocalizedString>
+        {
+            LocalizedString.English("Number")
```
Comment (ustims) [inline CloudFabric.EAV.Domain/Models/Attributes/TextAttributeConfiguration.cs:41]: This is a text attribute, is "Number" supposed to be here?

**Review by ustims — CHANGES_REQUESTED** [review]

File: CloudFabric.EAV.Domain/Models/Attributes/TextAttributeConfiguration.cs (hunk 2 above)
Comment (dgon-jd) [reply CloudFabric.EAV.Domain/Models/Attributes/TextAttributeConfiguration.cs]: Fixed

File: CloudFabric.EAV.Domain/Models/Attributes/TextAttributeConfiguration.cs (hunk 1 above)
Comment (dgon-jd) [reply CloudFabric.EAV.Domain/Models/Attributes/TextAttributeConfiguration.cs]: Fixed

**Review by ustims — APPROVED** [review]

**Pull request merged**